Source/Particles/Deposition/ChargeDeposition.H

```diff
@@ -81,10 +81,10 @@ void doChargeDepositionShapeN (const GetParticlePosition& GetPosition,
     const amrex::Real zmin = xyzmin[2];
 
     amrex::Array4<amrex::Real> const& rho_arr = rho_fab.array();
-    amrex::IntVect const rho_type = rho_fab.box().type();
+    //amrex::IntVect const rho_type = rho_fab.box().type();
```

Comment:
Could you remove any line that is commented out, if not needed anymore?
```suggestion
```
There will be similar suggestions throughout the code review.